tutorials/dlp-hybrid-inspect/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cloud Data Loss Prevention (DLP) Hybrid Inspect demonstration for SQL databases using JDBC
+description: Demonstrates how to inspect SQL table data using Cloud Data Loss Prevention with Hybrid Inspect.
+author: scellis,crwilson
+tags: Cloud DLP, Java, PII
+date_published: 2020-08-21
+---
+
+## Configuration and Build
+Depend in on the database you are connecting to, you may need to update the ```pom.xml``` file to include the proper JDBC client.
+
+__Compile everything__
```

Comment:
Give the reader some context about where they are running this command and what the prerequisites are.

Reply:
Added additional context Thoughts?